README.rst

```diff
@@ -386,6 +382,30 @@ The following means "reset all colors" (``0``), set foreground color to 18
 Constants are available via ``pdb.Color``, e.g. ``pdb.Color.red``
 (``"31;01"``), but in general any string can be used here.
 
+
+Example configurations
+^^^^^^^^^^^^^^^^^^^^^^
+
+An example configuration file, which activates ``sticky mode`` by default,
+would look like the following::
+
+    import pdb
```

Comment:
I think on master this should be more explicit and use `pdbpp` here.
(via #318)